File: Batch_22-26/Task-2/Siddharth_CS/twoactivity2/app/src/main/java/com/example/twoactivity/MainActivity.java
package com.example.twoactivity;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;

public class MainActivity extends AppCompatActivity {
     private EditText nameText,ageText;
     private Button button;
     private String name;
     private int age;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        nameText = findViewById(R.id.Personme);
        ageText = findViewById(R.id.PersonName2);
        button = findViewById(R.id.button);

        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                search();
            }
        });
    }
    public void search()
    {
       name = nameText.getText().toString().trim();
       age = Integer.parseInt(ageText.getText().toString().trim());

        Intent i = new Intent(MainActivity.this,activity1.class);

        i.putExtra(activity1.NAME,name);
        i.putExtra(activity1.AGE,age);

        startActivity(i);
    }
}

File: Batch_22-26/Task-3/Abhinav Rai/App_Task3/app/src/main/java/com/example/app_task3/MainActivity.kt
package com.example.app_task3

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val Actbut1 = findViewById<Button>(R.id.Visit)
        Actbut1.setOnClickListener{
            val Intent = Intent ( this,MainActivity2::class.java)
            startActivity(Intent)
        }
    }
}
